src/views/Pools/components/PoolCard/AprRow.tsx

```diff
@@ -35,25 +27,32 @@ const AprRow: React.FC<AprRowProps> = ({
 
   const { targetRef, tooltip, tooltipVisible } = useTooltip(tooltipContent, { placement: 'bottom-end' })
 
-  const earningTokenPrice = useGetApiPrice(earningToken.address ? getAddress(earningToken.address) : '')
+  const earningTokenFarmForPriceCalc = useFarmFromTokenSymbol(earningToken.symbol)
+  const earningTokenPrice = useTokenPriceBusd(earningTokenFarmForPriceCalc.pid)
+  const earningTokenPriceAsNumber = earningTokenPrice && earningTokenPrice.toNumber()
+
+  const stakingTokenFarmForPriceCalc = useFarmFromTokenSymbol(stakingToken.symbol)
+  const stakingTokenPrice = useTokenPriceBusd(stakingTokenFarmForPriceCalc.pid)
+  const stakingTokenPriceAsNumber = stakingTokenPrice && stakingTokenPrice.toNumber()
```

Comment:
This is a bit icky. I started this work by tackling the smallest pieces first, but I may have been better off tackling the biggest first (i.e. storing farm prices to state).

In the next PR I will be tackling the last of the API uses - the values stored to farms' prices state. 

I ought to then be able to tidy up price retrieval throughout the rest of the app, simplifying this sort of thing.